compel/include/uapi/infect-util.h

```diff
@@ -7,7 +7,7 @@
  * compel_run_id is a unique value of the current run. It can be used to
  * generate resource ID-s to avoid conflicts with other processes.
  */
-extern uint64_t compel_run_id;
+extern char compel_run_id[37];
```

Comment:
Would it be possible to define `37` somewhere as a constant?

Reply:
added a define